packages/@react-types/listbox/package.json

```diff
@@ -6,7 +6,7 @@
   "types": "src/index.d.ts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/adobe-private/react-spectrum-v3"
+    "url": "https://github.com/adobe/react-spectrum"
```

Comment:
I noticed this and menu don't have readme.md files. I assume they do. Is that a different issue or should I fix that here? I didn't check if there are other missing READMEs.